xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1044 @@
+      <title>OBS Workflows</title>
+
+      <para>An SCM event occurs and OBS receives the corresponding webhook.
+      Now is when the <emphasis role="bold">OBS workflows</emphasis> come into
+      play.</para>
+
+      <para>A workflow is nothing else than a sequence of <emphasis
+      role="bold">steps</emphasis> or actions the user wants to perform in
+      OBS. The user can describe the steps to run in a YAML configuration
+      file.</para>
+
+      <para>To do so, in the root directory of your GitHub repository, create
+      a directory <emphasis>.obs</emphasis> which contains a file called
+      <emphasis>workflows.yml</emphasis>.</para>
+
+      <para>The content of <emphasis>.obs/workflows.yml</emphasis> could look
+      like this:<screen>main_workflow:
+  steps:
+    - branch_package
+        source_project: OBS:Server:Unstable
+        source_package: obs-server
+        target_project: OBS:Server:Unstable:CI
+  filters:
+    event: pull_request
+rebuild_master:
+  steps:
+    - rebuild_package
+        project: home:Admin
+        package: ctris
+  filters:
+    event: push
+    branches:
+      only:
+        - master</screen></para>
+
+      <para>The file can contain multiple workflows, but each one needs an
+      <emphasis>unique</emphasis> name. The previous example contains two
+      workflows: <emphasis>main_workflow</emphasis> and
+      <emphasis>rebuild_master</emphasis>.</para>
+
+      <para>OBS assumes the configuration file is placed in your target branch
+      and not in other branches or forks.</para>
+
+      <note>
+        <title>Workflows Cleanup</title>
+
+        <para>Workflows are cleaned up after themselves when pull requests are
+        closed or merged. Projects in which you branched or linked packages
+        will be deleted, the same thing for the repositories you configured
+        for those projects.</para>
+
+        <para>If you decide to reopen the pull request, this will also be
+        handled by workflows. Your projects, their branched/linked packages
+        and their configured repositories will be restored.</para>
+      </note>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.steps">
+        <title>Steps</title>
+
+        <para>Currently, the supported steps are:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Branch a Package (branch_package).</para>
+          </listitem>
+
+          <listitem>
+            <para>Link a Package to a Project (link_package).</para>
+          </listitem>
+
+          <listitem>
+            <para>Configure Repositories/Architectures for a Project
+            (configure_repositories)</para>
+          </listitem>
+
+          <listitem>
+            <para>Rebuild a Package (rebuild_package)</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Some of these steps need to get the sources from the SCM pull
```

Comment:
None of the steps *need* this. For some of the steps it makes sense for some of the SCM events. Like `branch_package` and `pull_request`. I think this is something for the use case chapter.